src/lua_details.cc

```diff
@@ -114,21 +114,39 @@ void check_script_timeout(lua_State* lua_state)
 std::variant<sol::object, std::string>
 execute_lua_script_safely(sol::state& lua, sol::string_view script)
 {
+    const std::string anchor = u8"\u2693";
+    constexpr gul14::string_view chunk_prefix{ u8"[string \"\u2693\"]:" };
```

Comment:
I think it is not very nice to have the anchor thing in to independent definitions.

Reply:
> I think it is not very nice to have the anchor thing in two independent definitions.

Originally I built the `chunk_prefix` with `cat()` at runtime... but then I had Jason in the back of my head shouting "constexpr all the things"... :)

Well, I want a `constexpr` string. Until then, I guess all other solutions for this are just as ugly.

Reply:
Another point **for** your previoius solution...

With the code above you encode `"[string "` and `"]:"` _also_ with UTF8, while in the code that generates the string (i.e. Lua) it is encoded 'normally' (i.e. implementation defined). In general this is the same, especially mainstream desktop machines, but then do not need this anyhow.

Correct would be to have `anchor` as UTF8 string and cast that to `char*` and insert it into an implementation defined encoding string.

Another maybe better approach would be to
> Change u8 literals to ordinary literals with escape sequences

especially as we just use them as magic numbers and not some concrete characters. It is rather easy to put the UTF8 codes as bytes in the 'normal' string.
(see https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1423r2.html)

This problem does not arise (except the `char8_t` stuff with C++20) with our other magic character (ABORT), because it is always incoded just on its own.

Reply:
Either the code is correct or it is UB, and in this case it is the later.

Maybe my problem is that I dislike the unicode codepoints anyhow, and so the `\uxxxx` is just another way to add random bytes in the string, in this case `e2` `9a` `93`.